libs/api/domains/cms/src/lib/models/organizationSubpage.model.ts

```diff
@@ -26,6 +26,12 @@ export class OrganizationSubpage {
   @Field(() => [SliceUnion], { nullable: true })
   slices: Array<typeof SliceUnion>
 
+  @Field({ nullable: true })
+  sliceCustomRenderer: string
+
+  @Field({ nullable: true })
+  sliceExtraText: string
```

Comment:
Should these fields maybe be optional since they are  nullable?
```suggestion
  @Field({ nullable: true })
  sliceCustomRenderer?: string

  @Field({ nullable: true })
  sliceExtraText?: string
```